tests/testsuite/js_library.rs

```diff
@@ -0,0 +1,138 @@
+use ansi_term::Color;
```

Comment:
New module tests are now kept alongside the module they test (e.g. https://github.com/starship/starship/blob/master/src/modules/nodejs.rs#L38)

Reply:
no worries, I'll get it rebased and then move it around :)